react/hooks/useGetBagsSgrIDs.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+
+const useGetBagsSgrIDs = () => {
+  const [bagsIDs, setBagsIDs] = useState([''])
```

Comment:
Nu prea vad motivul pentru care il initializezi cu array care are empty string. De ce mai adaugi si empty string?

Reply:
Pt ca altfel primesc la set state: Argument of type 'string[]' is not assignable to parameter of type 'SetStateAction<never[]>'

Reply:
Am pus asa ... = useState<string[]>() si e ok acucm